Card.cpp

```diff
@@ -1,4 +1,6 @@
 #include "card.h"
+#include <chrono>
+#include <thread>
 
 void Card::initialize(Ui::MainWindow *ui_ptr)
```

Comment:
Is this considered a constructor?
If so, should we have a destructor in this file also?